new-dti-website/src/app/apply/apply-timeline.tsx

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable no-param-reassign */
+
+'use client';
+
+import React, { useState } from 'react';
+
+const ApplyTimeline: React.FC = () => {
+  const stages = [
+    {
+      title: 'APPLICATIONS OPEN!',
+      content:
+        'We’re welcoming any and all students who are looking to make a difference through tech. Applicants are not considered on a rolling basis.',
+      date: '2024-03-14',
+      location: 'Online Submission'
+    },
+    {
+      title: 'INFORMATION SESSION 1',
+      content:
+        'Come join us for a session to learn about DTI, our goals, and our subteams. There will be time to chat with DTI members of all roles after!',
+      date: '2024-03-18',
+      location: 'Goldwin Smith Hall G26'
+    },
+    {
+      title: 'INFORMATION SESSION 2',
+      content:
+        'We’re welcoming any and all students who are looking to make a difference through tech. Applicants are not considered on a rolling basis.',
+      date: '2024-03-19',
+      location: 'Kimball Hall B11'
+    },
+    {
+      title: 'APPLICATIONS DUE!',
+      content:
+        'We’re welcoming any and all students who are looking to make a difference through tech. Applicants are not considered on a rolling basis.',
+      date: '2024-04-01',
+      location: 'Online Submission'
+    }
+  ];
```

Comment:
For the other pages, we put these large objects in a separate json file in the same directory. Consider doing this too -- it'll make editing application timelines easier in the future, too!